recipes/the-silver-searcher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "the-silver-searcher" %}
```

Comment:
Should we use `-` or `_` given the repo uses `_`? Are there instances of this getting packaged in Homebrew or Linux distros we can look for some perspectives on this?